content/blog/blog-post-08.md

```diff
@@ -0,0 +1,52 @@
+
+Historically the HPC community has been characterized by a large amount of fragmentation, with many similar or even identical capabilities developed at different institutions.
+The lack of reliable software support funding mechanism in the scientific community led to a reluctance of relying on third party libraries, and the lack of community governance models hindered the establishment of joint multi-institutional development efforts.
+
+For Kokkos the US Department of Energy's (DOE) Exascale Computing Project changed that.
+By providing a framework that included all DOE's national laboratories it was possible to grow the Kokkos team beyond a single institution and establish a multi-laboratory collaborative effort to develop a robust performance portability solution.
+
+HPSF intents to be the common ground for establishing these collaborations on a global scale. 
+ 
+# HPSF Structure
+
+HPSF will be controlled by a Governing Board and a Technical Advisory Council.
+The Governing Board will consist of a mix of representatives from HPSF member organizations and some representation from technical projects such as Kokkos.
+It decides how to allocate the budget of the foundation, and also controls marketing and HPSF events.
+Furthermore, the Governing Board decides on general HPSF rules.
+
+The Technical advisory committee organizes technical working groups and develops criteria for the project lifecycle in HPSF. It consists mainly of representatives of the technical projects.
+Technical governance of projects associated with HPSF is still up to each individual project.
+The Advisory Council is also the primary body to organize day-to-day collaboration with other Linux Foundation efforts.
+
+{{< image src="img/blog/2024/hpsf-structure.png" style="float: center; height=10">}}
+
+
+# Open for Global Business
+
+Key benefits of belonging to the High-Performance Software Foundation include open governance models, resources for improving software quality and building community, as well as collaborations with the larger Linux computing ecosystem.
+Leveraging well established open governance models from the Linux Foundation, HPSF provides a framework to organize multi-institutional collaborations.
+An institution that wants to leverage Kokkos and also become contributors can now rest assured that the project is not just subject to the whims of some project leads paid for by DOE.
+Donating rights to Kokkos to the Linux Foundation also demonstrates that the Kokkos team and DOE has been serious with their intent to evolve Kokkos into a true community project.
+
+# Benefits for Kokkos Users & Developers
+
+Kokkos developers and users will likely experience, few, if any, change in their day-to-day activities.
+It will be developed using the existing mechanisms on github with funding by the existing stake-holders.
+On the legal side the trademark for Kokkos is now held by the Linux Foundation.
```

Comment:
Should we mention or link to any rules kokkos-related projects need to follow? Kind of like we did in the release briefing